crates/Cargo.toml

```diff
@@ -10,7 +10,7 @@ members = [
 resolver = "2"
 
 [workspace.package]
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
Please do not change the crate versions in your PR. We make those changes in separate release PRs (and as above, this is not a backwards-compatible change).

Reply:
reverted